scripts/lib/libvirt/test_libvirt_setup.py

```diff
@@ -120,31 +109,20 @@ class TestLibvirtComputeConfig(unittest.TestCase):
 
     def test_compute_config(self):
         args = arg_parse(
-            ["cloud",
-             "nodecounter",
-             "macaddress",
-             "controller_raid_volumes",
-             "cephvolumenumber",
-             "drbdserial",
-             "computenodememory",
-             "controllernodememory",
-             "libvirttype",
-             "vcpus",
-             "emulator",
-             "vdiskdir"])
-        args.cloud = "cloud"
-        args.nodecounter = "1"
-        args.macaddress = "52:54:01:77:77:01"
-        args.controller_raid_volumes = 0
-        args.cephvolumenumber = "1"
-        args.computenodememory = "2097152"
-        args.controllernodememory = "5242880"
-        args.libvirttype = "kvm"
-        args.vcpus = "1"
-        args.emulator = "/usr/bin/qemu-system-x86_64"
-        args.vdiskdir = "/dev/cloud"
-        args.drbdserial = ""
-        args.bootorder = "3"
+            [("cloud", str, "cloud"),
```

Comment:
I would completely drop `arg_parse` and use something like this:

```python
class Arguments(object):
    pass

...

args = Arguments()
args.nodecounter = 1
args.macaddress = '52:54:01:77:77:01'
....
```

Also I will do that in a different PR, because is really unrelated with this (argparse problem in the tests is not breaking HA, nor the tests, even is wrong)